RELEASE v1.12.0 (#26)
hms-networks/sc-ewon-flexy-extensions-lib: CHANGELOG.md

# Ewon Flexy Extensions Library Changelog

## Version 1.12.0
### Features

> - Added additional known status codes in MqttStatusCode class

Reviewer: ```suggestion
- Added additional status codes in MqttStatusCode class
```

Author: Updated

> - Added SmartMqttManager wrapper of MqttManager to handle common use cases

Reviewer: ```suggestion
- Added ConstrainedMqttManager.java [a wrapper of MqttManager.java] to handle common MQTT use cases
```

Author: Updated

> - Added SCHttpUtility methods to output response data to a file

Reviewer: ```suggestion
- Added SCHttpUtility methods to output HTTP request response data to a file
```

Author: changed

### Bug Fixes

> - Fixed improper Javadoc in AutomaticRetryCodeLinear

Reviewer: ```suggestion
- Fixed improper Javadoc in AutomaticRetryCodeLinear.java
```

Not sure if we should include file extensions but it seems to clarify the fact that we are referencing a filename.

Author: Updated

### Other

> - Simplified MqttManager by internally getting status and passing to implementation methods 

Reviewer: ```suggestion
- Simplified MqttManager.java by internally getting status and passing to implementation methods 
```

Author: Updated
